Sources/Jobs/JobExecutionOptions.swift

```diff
@@ -0,0 +1,85 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Hummingbird server framework project
+//
+// Copyright (c) 2024 the Hummingbird authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See hummingbird/CONTRIBUTORS.txt for the list of Hummingbird authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import struct Foundation.Date
+
+/// Job execution type
+public struct JobExecutionOptions: ExpressibleByArrayLiteral, Sendable {
+    /// Job execution options
+    public struct Option: Sendable {
+        internal enum _Option {
+            case delayed(until: Date)
+            case unknown
```

Comment:
What is unknown for?

Reply:
I think it's remnants when I was learning about `ExpressibleByArrayLiteral`. It's not needed since an empty array can set as default.